autoscrub/version_check.py

```diff
@@ -0,0 +1,53 @@
+import six
+import re
+from distutils.version import LooseVersion
+
+if six.PY3:
+    from subprocess import Popen, PIPE
+else:
+    # backported subprocess module
+    from subprocess32 import Popen, PIPE
+__terminal_encoding = "utf-8"
+
+
+def get_ffmpeg_version(strip_prefix=True):
+    """Gets the version of ffmpeg, stripping prefix comprising alpha/dash
+    characters, e.g. nightly build 'N-' prefix.
+    Returns None if version cannot be determined."""
+    command = ["ffmpeg", "-version"]
+    p = Popen(command, stdout=PIPE, stderr=PIPE)
```

Comment:
I think this will be more reliable across platforms/Python versions if we use `autoscrub._agnostic_Popen`.